include/cc_gpu_sketch_alg.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "cc_sketch_alg.h"
+#include "cuda_kernel.cuh"
+
+class CCGPUSketchAlg : public CCSketchAlg{
+private:
+  CudaUpdateParams** cudaUpdateParams;
+  //Sketch** sketches;
+  long* sketchSeeds;
```

Comment:
Since GPU cannot call sketch's `get_seed()` function within GPU kernel, I collect every sketch's seeds ahead of time, store them in `sketchSeeds`, then transfer it to GPU so the kernel can read it during sketch updates.

Reply:
I see that makes sense. It should probably be sufficient to pass in the single seed that defines all the others. That is, the SketchSeed is just slightly modified each time to compute the various column seeds.

Reply:
Yup, I agree. I have applied the change in this commit: https://github.com/MangoShip/GraphZeppelin/pull/3/commits/dbf4224d63d5a42974bbb617e6db2bd352a84124